### ISSUE TITLE
Provide option to save/read neural network to/from file

### ISSUE THREAD

**Issue by wgrundlingh** [bips-hb/neuralnet#23]
Perhaps you've trained a neural network and you don't want to re-train it within some chunk of code (to save time). It would be great if there was a method to save the neural network to file and then just read it for use later.

This would be especially helpful when using `neuralnet` inside an Rmarkdown document or presentation, since the document has to be knit together in its entirety every time, retraining the network with every compilation. Reading from file would improve the speed of compilation.

**Comment by mnwright** [comment]
Why not simply use R's `saveRDS()` and `readRDS()`?

**Comment by wgrundlingh** [comment]
Of course... forgot about that. How about adding a function as a wrapper for `saveRDS()` and `readRDS()`? Even if that's all it is, just a wrapper.
